### ISSUE TITLE
TFL platform changes breaks module

### ISSUE THREAD

**Issue by charliedowler** [bencevans/node-oyster#4]
Hey,

TFL has yet again, broken this module :-(

It's returning this as the body:

```html
<html>
    <head>
    <title>Object moved</title>
    </head>
    <body>
        <h2>Object moved to <a href="/Oyster/Login/TransferOyster">here</a>.</h2>
    </body>
</html>
```

Might be related to the new [contactless](https://contactless.tfl.gov.uk/) app, it links you back to the old oyster management page. Probably preparing for a new platform release.

I couldn't find a temporary solution. I'll give it another shot when I have some time.

Cheers

**Comment by bencevans** [comment]
Afternoon. I've written a fix (https://github.com/bencevans/node-oyster/commit/c34c87b73ff541a651b9ce6e0e0c2bc65b2cf74c) and have published on npm in `v0.2.0`.

Happy New Year Oyster Fans!

**Comment by charliedowler** [comment]
Awesome! Thanks
